application/sbsk-integration-test/build.gradle

```diff
@@ -18,5 +18,5 @@ dependencies {
 }
 
 test {
-	exclude 'com/sbsk/integration/**'
+	exclude 'com/sbsk/integration/integration/**'
```

Comment:
Perhaps we can modify the package so as for the path to be `com/sbsk/integration/**`